### PR TITLE
Databases/week2

### PR DESCRIPTION
**Pull request by Gohar33**

### CODE REVIEW

**Review by Elia-Git — COMMENTED** [review]
Please find all my comments and try to apply them and let me see you changes before merging if you can!

File: databases/week2/queries.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
+use hyf_lesson2;
+
+SELECT user.name, user.email, task.title, task.description, user_task.task_id
+FROM task
+JOIN user 
+ON user.id=task.id
+JOIN user_task
+ON user.id = user_task.user_id
+WHERE email LIKE "%@spotify.com%";
```
[the remark this answers is not in the document]
Comment (Gohar33) [reply databases/week2/queries.sql:9]: Hi Elias, you're right, I get 4 similar tasks :(

File: databases/week2/bookdata.sql [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+-- book_genre table
+
+INSERT into book_genre (book_id,genre_id) values(1, 1);
+INSERT into book_genre (book_id,genre_id) values(2, 1);
+INSERT into book_genre (book_id,genre_id) values(3, 2);
+INSERT into book_genre (book_id,genre_id) values(4, 3);
+INSERT into book_genre (book_id,genre_id) values(5, 3);
+INSERT into book_genre (book_id,genre_id) values(6, 2);
+
+
```
[the remark this answers is not in the document]
Comment (Gohar33) [reply databases/week2/bookdata.sql:85]: I thought at the beginning that should insert data and sincerely only after inserting I finally understood the relationship between tables. Then when I submitted the homework I realized there was no need for that. But it helped a lot.

File: databases/week2/queries.sql [hunk 3]
```diff
@@ -17,7 +15,7 @@ WHERE user.name LIKE "%Donald Duck%"
 AND status.name LIKE "%Not started%";
 
 -- Get all the tasks for 'Maryrose Meadows' that were created in september
-SELECT user.name, task.title, status.name AS status_name
+SELECT user.name, task.title, status.name AS task_created
```
Comment (Elia-Git) [inline databases/week2/queries.sql:18]: I meant this should be 
`SELECT user.name, task.title, task_created`

**Pull request merged**